packages/push/src/PushRegistration.ts

```diff
@@ -0,0 +1,82 @@
+
+import { AeroGearConfiguration, INSTANCE, isCordovaAndroid, isCordovaIOS } from "@aerogear/core";
+import axios from "axios";
+
+declare var window: any;
+declare var document: any;
+
+/**
+ * AeroGear UPS registration SDK
+ *
+ * Usage:
+ * // Initialize SDK first
+ * app.init(config);
+ * let registration = new PushRegistration();
+ * registration.register("myAppleOrFirebaseToken");
+ */
+export class PushRegistration {
+
+  public static readonly TYPE: string = "push";
+  public static readonly API_PATH: string = "rest/registry/device";
+
+  private pushConfig: any;
+
+  constructor() {
+    const configuration = INSTANCE.getConfigByType(PushRegistration.TYPE);
+    if (!configuration || configuration.length === 0) {
+      console.warn("Push configuration is missing. UPS server registration will not work.");
+      this.pushConfig = {};
+    } else {
+      this.pushConfig = configuration[0];
```

Comment:
I really don't like this option, I see we also have this on other platforms, IMO the developer should provide the ID of the push he/she wanna use.

Reply:
@danielpassos This behavior was agreed on mailing list. Feel free to post this there. We cannot change this without changing other SDK's etc.